README.md

```diff
@@ -37,22 +37,41 @@ pip install bespokelabs-curator
 
 ```python
 from bespokelabs import curator
-import os
+from datasets import Dataset
+from pydantic import BaseModel, Field
+from typing import List
 
-os.environ['OPENAI_API_KEY'] = 'sk-...' # Set your OpenAI API key here
+# Create a dataset object for the topics you want to create the poems.
+topics = Dataset.from_list([{"topic": "Dreams vs. reality"},
+                            {"topic": "Urban loneliness in a bustling city"},
+                            {"topic": "Beauty of Bespoke Labs's Curator library"}])
```

Comment:
Change to
```python
topics = Dataset.from_dict({"topic": [
    "Dreams vs. reality",
    "Urban loneliness in a bustling city",
    "Beauty of Bespoke Labs's Curator library"
]})
```

Just a formatting nit, perhaps people like `from_dict` better? To illustrate it's a one column with three rows. Repeating "topic" key doesn't look good code, may give ppl bad impression.

Reply:
Done. I felt the from_list method is a bit clearer to show the rows created.
But this is fine.